tools/tikv-control.md

```diff
@@ -103,6 +105,23 @@ key: zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\00
 
 In this command, the key is also the escaped form of raw key.
 
+### Recover from MVCC data corruption
```

Comment:
I prefer moving this with  `Force Region to recover the service from failure of multiple replicas` to the bottom